base_layer/wallet_ffi/README.md

```diff
@@ -67,10 +79,12 @@ Install [Rust](https://www.rust-lang.org/tools/install)
 
 Install the following tools and system images
 ```Shell Script
-rustup toolchain add nightly-2019-10-04
-rustup default nightly-2019-10-04
+rustup toolchain add nightly-2020-01-08
+rustup default nightly-2020-01-08
 rustup component add rustfmt --toolchain nightly
 rustup component add clippy
+rustup target add x86_64-apple-ios aarch64-apple-ios # iPhone and emulator cross compiling
+rustup target add x86_64-linux-android aarch64-linux-android armv7-linux-androideabi # Android device cross compiling
```

Comment:
I have noticed that whenever rust updates itself it invalidates these and they need to be run again.

These lines also appear in the build script.

Reply:
Perhaps they're positioned in the script in a way that they don't get executed in the right place. Until I manually ran these commands, the FFI build would fail

Reply:
They are on lines 57 and 96 respectively of `mobile_build.sh`.
So they will always be run just before building for the platform they are needed.

Will need to investigate it further.